CorpusSearch/ClientApp/src/routes/Newbies.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {useEffect, useState} from "react"
+function CorpusLink({ident="",name=""}): JSX.Element {
+    if ( name.slice(0,2) != "🎥"){
+        name = "📖" + name
+    }
+    return <div>
+        <a href={"docs/" + ident}>{name}</a>
+    </div>
+}
+export const Newbies = () => {
+type DocType =  {name:string, ident:string, uploaded:string}
+const [newDocs, setNewDocs  ] 
+    = useState<DocType[]>([])
+useEffect(()=>  {
+    const fetchNewDocs = async () => {
+        await fetch("api/metadata/latest")
+            .then((res)=> {
+                if (!res.ok) {
+                    console.warn(`metadata/latest returned status: ${res.status}`)
+                }
+                return res.json()
+            }).then((json: DocType[])=> {
+                setNewDocs(json)
+        })
+    }
+    fetchNewDocs().catch((e:Error)=> console.warn(`fetchNewDocs returned error:" ${e.message}`))
+},[]
+    )
+    const len = newDocs.length
+    if (!len) return <></>
+    const newarray: JSX.Element[] = []
+    for ( let i=0; i < len; i++) {
+        const {name,ident} = newDocs[i]
+        const link = <CorpusLink key={name} name={name} ident={ident} />
+        newarray.push(link)
+    }
+    return (<div className="Newbies">
+        <header className="Newbies-header">
+        </header>
+        <div>New and Featured Transcriptions:<br/>
```

Comment:
```suggestion
        <div>Recent Additions:<br/>
```

Reply:
Fine. This was a placeholder name anyway. Only it's not just additions but also changes

Reply:
It should ideally be additions, otherwise an automated edit would mean that all files are listed

Reply:
Done although I changed the wording slightly